PyInstaller/depend/analysis.py

```diff
@@ -128,15 +128,20 @@ def _msg(self, level, s, *args):
 
         msg = "%s %s" % (s, ' '.join(map(repr, args)))
 
-        try:
-            fn, lno, func, sinfo = logger.findCaller()
-        except ValueError:  # pragma: no cover
-            fn, lno, func, sinfo = "(unknown file)", 0, "(unknown function)", None
-
         if is_py2:
+            try:
+                fn, lno, func = logger.findCaller()
+            except ValueError:  # pragma: no cover
+                fn, lno, func = "(unknown file)", 0, "(unknown function)", None
```

Comment:
If this commit is outdated, why is it shown in https://github.com/pyinstaller/pyinstaller/pull/2433/commits then?